code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -142,12 +145,20 @@
 	var/enthralling = 0
 
 /obj/effect/proc_holder/spell/targeted/enthrall/cast(list/targets)
+	var/thrallsPresent = 0
+	var/mob/living/carbon/human/user = usr
+	for(var/datum/mind/mindToCount in ticker.mode.thralls)
+		thrallsPresent++
```

Comment:
^ this but make sure to call listclearnulls() on ticker.mide.thralls BEFORE counting the len.
a list of 30 nulls is still 30 long!